components/community/Card.tsx

```diff
@@ -0,0 +1,113 @@
+import Link from 'next/link';
+import React from 'react';
+
+import { HeadingLevel, HeadingTypeStyle } from '@/types/typography/Heading';
+import { ParagraphTypeStyle } from '@/types/typography/Paragraph';
+
+import IconArrowUp from '../icons/ArrowUp';
+import Heading from '../typography/Heading';
+import Paragraph from '../typography/Paragraph';
+
+interface SmallHomeCardProp {
+  icon: string;
+  tagline: string;
+  taglineBg: string;
+  type?: string;
+  heading: string;
+  description: string;
+  bg: string;
+  btnText: string;
+  btnBg: string;
+  link: string;
+};
+
+/**
+ * @description This component displays Small Home Card.
+ * @param {SmallHomeCardProp} props - The props for Small Home Card component.
+ * @param {string} props.icon - The icon for the card.
+ * @param {string} props.tagline - The tagline for the card.
+ * @param {string} props.taglineBg - The background color for the tagline.
+ * @param {string} props.type - The type of the card (either 'small' or 'large').
+ * @param {string} props.heading - The heading for the card.
+ * @param {string} props.description - The description for the card.
+ * @param {string} props.bg - The background color for the card.
+ * @param {string} props.btnText - The text for the button.
+ * @param {string} props.btnBg - The background color for the button.
+ * @param {string} props.link - The link for the button.
+ */
+export default function SmallHomeCard({
```

Comment:
Why the component has been named as SmallHomeCard?

Reply:
This name is used in the master branch. But I think it will be more accurate to name it `Card`. Also, it's not used anywhere in the master branch except for writing its own test.